tests/foreman/cli/test_domain.py

```diff
@@ -113,9 +114,14 @@ def valid_delete_params():
     ]
 
 
-class DomainTestCase(CLITestCase):
+class TestDomain:
     """Domain CLI tests"""
 
+    @pytest.fixture(scope="class")
+    def class_domain(self):
+        """Create shared domain"""
+        return make_domain()
```

Comment:
How would you like to try stripping these out of the class entirely and making this a module-level fixture?